intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -128,11 +129,15 @@ We might say...
 
 ### State and Behavior
 
-When we start defining our own classes, we will specify their **state** and their generic **behavior.**
+When we start defining our own classes, we will specify their **state** and their generic **behavior**.
+
+**State**, **Attribute**, **Behavior**, and **Method** are all new terms we haven't heard before, but don't be intimidated by them! 
+
+**State** is just a term that means "all the **attributes** of an object". **Attribute** is really just a specific type of **variable** defined within a class. 
 
-**State** describes any data or attributes that should be kept over its lifetime—from the moment we create an instance until we let the system reclaim its resources.
+These **attributes** will be kept over the object's lifetime—-from the moment we create an instance until the object goes out of scope.
 
-**Behavior** describes any actions that objects of this class should be able to do.
+**Behavior** describes any actions that objects of this class should be able to do. In other words, **behavior** is just a term that means "all the **methods** of a class". **method** is really just a specific type of **Function** defined within a class.
```

Comment:
I can remove "just" but I find this new wording much harder to read, as its using passive instead of active voice.

Reply:
Could we add some flow to it? Possibly

```suggestion
**Behavior** describes any actions that objects of this class should be able to do.

To give classes _behavior_, we define functions inside of a class. A **method** is the name for any function that's defined inside of a class. In other words, _behavior_ is a term that refers to all methods of a class.
```

Reply:
What do you mean by "flow"? This feels like is flows way less well to me.

Reply:
The phrase "In other words" just feels really abrupt to me in this context? There's one sentence of a definition previously before; if we need to say the phrase "In other words" in the second sentence, could we just change the first sentence?

The phrase "Method is really just a specific type of function" comes across as really odd and abrupt to me-- the first time we use the word "method" in this lesson is in the vocab table, a couple sentences up that says, "don't be intimidated by this vocab," and in the sentence right before it; we don't need to use words to make a reveal about what methods are if we can define it in the sentence before. I get that it's trying to be very comforting, but couldn't we simplify the language and sentences? I feel like this kind of comfort comes across really well conversationally, but comes across as a little bit strange as text.

The flow is very "this is what behavior is. But really, here's a better definition of behavior that includes methods. But you've never heard of methods before, methods are actually just something else, even though you don't have a different definition of methods already."." It feels like the thoughts are trying to one-up each other immediately afterwards. I think that there are different ways to simplify it or to add comforting logic that doesn't feel very "don't give as much weight to the definition presented in the sentence immediately prior because we're going to give a simpler definition." I think that this pattern of giving a definition and then giving a second, summary definition is better for longer pieces, not necessarily one sentence after another; it feels like it emotionally escalates this paragraph to "the sentence before was tough but here's a second better sentence," as opposed to "here are two ways to define this new term"

Open to other suggestions, I just think the current wording adds chaos in one paragraph. The second sentence feels like a reveal to the first sentence if we use "In other words," and the third sentence sounds like someone trying to give a quick definition of methods and say methods aren't a big deal but no plain definition of methods has come up yet here.